.travis.yml

```diff
@@ -13,22 +13,22 @@ env:
 # ------------------------
 jobs:
   include:
-    - stage: Tests
-      name: Dead link check
-      if:
-        type = pull_request OR type = push AND branch =~ /^master|[0-9]+-(dev|stable)$/
-        OR type = cron
-      language: node_js
-      node_js: 12
-      before_script:
-        - npm ci
-        - npm run doc-prepare
-        - $(npm bin)/kuzdoc iterate-repos:install --repos_path doc/framework/.repos/
-        - $(npm bin)/kuzdoc framework:link -d /sdk/java/3/ -v 3
-
-      script:
-        - gem install typhoeus
-        - cd doc/framework/ && HYDRA_MAX_CONCURRENCY=20 ruby .ci/dead-links.rb -p src/sdk/java/3/
+    # - stage: Tests
```

Comment:
This section is commented, is this normal?

Reply:
Yes
Since the new Kuzzle documentation which is cloned for the test is the new one, the deadlink check will fail until we change everything (every link, and have the actual new doc). So we decided to comment this test for the meantime